### PR TITLE
Adds missing report documentations

### LINKED ISSUE #16792: Have a report description for every report
We should click through the UI and should make sure to have a report description for every report shown in the UI. This is currently for example not the case for Device reports and some other reports
image

### PR DESCRIPTION
**Pull request by sgiehl**
### Description:

This also fixes an issue where the report documentation was not visible after switching to a related report.

fixes #16792

### Review

* [ ] Functional review done
* [ ] Usability review done (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] Security review done [see checklist](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] Code review done
* [ ] Tests were added if useful/possible
* [ ] Reviewed for breaking changes
* [ ] Developer changelog updated if needed
* [ ] Documentation added if needed
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by sgiehl** [comment]
Would be good if someone else could look through the documentations I've added and maybe improve them further. Didn't spent too much time on thinking of good descriptions.

Also some of the dynamically created reports currently don't have a documentation, like the custom dimension reports. Not sure if we should add one there as well.

And we need to check all our plugins as well maybe

**Comment by diosmosis** [comment]
@mattab maybe we could have Daniel, the person rewriting our technical guides, to look over these docs later? For now we could merge after checking for small errors, then they could look through the UI and make tweaks to the report documentation w/o having to look through code. What do you think?

File: plugins/Goals/lang/en.json [hunk 1]
```diff
@@ -108,6 +109,7 @@
         "GoalsAdjective": "Goals %s",
         "VisitPageTitle": "Visit a given Page Title",
         "VisitsUntilConv": "Visits to Conversion",
+        "VisitsUntilConvReportDocumentation": "This reports show how many visits a visitor need to convert a goal.",
```
Comment (Findus23) [inline plugins/Goals/lang/en.json]:
```suggestion
        "VisitsUntilConvReportDocumentation": "This reports show how many visits a visitor needs to convert a goal.",
```

**Review by Findus23 — COMMENTED** [review]
just some simple spellchecking

**Comment by diosmosis** [comment]
I think spending time making the report docs perfect here is probably time ill-spent, so I'll do some quick checks for typos and create a new issue to revise the report docs in the future. If that must be done in 4.1, it can be prioritized there.

**Comment by tsteur** [comment]
Be great to do it as part of 4.1 👍

**Comment by diosmosis** [comment]
@sgiehl I made some revisions to some of the docs, can you check if they sound better to you? Feel free to revert any changes I made.

**Comment by sgiehl** [comment]
@diosmosis the changes look good. If noone else does one to take a look, it might be good to merge.

**Pull request merged**